account_bank_statement_import_camt_oca/tests/test_import_bank_statement.py

```diff
@@ -0,0 +1,123 @@
+"""Run test to import camt.053 import."""
+# © 2013-2016 Therp BV <http://therp.nl>
+# Copyright 2017 Open Net Sàrl
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import base64
+import difflib
+import pprint
+import tempfile
+
+
+from odoo.tests.common import TransactionCase
+from odoo.modules.module import get_module_resource
+
+
+class TestParser(TransactionCase):
+    """Tests for the camt parser itself."""
+    def setUp(self):
+        super(TestParser, self).setUp()
+        self.parser = self.env['account.bank.statement.import.camt.parser']
+
+    def _do_parse_test(self, inputfile, goldenfile):
+        testfile = get_module_resource(
+            'account_bank_statement_import_camt_oca',
+            'test_files',
+            inputfile,
+        )
+        with open(testfile, 'rb').read() as data:
```

Comment:
```with open(testfile, 'rb') as handle:
    data = handle.read()
```

Reply:
Ugh... monday mornings. Sorry about that, correction coming up.